infomed/settings.py

```diff
@@ -77,8 +78,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.contrib.gis.db.backends.postgis',
+        'NAME': 'gis',
+        'USER': 'user001',
+        'PASSWORD': '123456789',
```

Comment:
I know that for now, it's OK to keep dummy data as this here, but I'd rather prefer we don't do this at all so that nobody is even tempted to put credentials in the codebase. Is there a way to use environment for this? Maybe a `.env` file?

Reply:
Done.